Makefile

```diff
@@ -121,6 +121,12 @@ test: manifests generate fmt vet envtest ## Run tests.
 build: manifests generate fmt vet ## Build manager binary.
 	go build -o bin/manager cmd/main.go
 
+.PHONY: vendor
+vendor:
+	go mod tidy
+	go mod vendor
+	go mod verify
```

Comment:
This should be in a separate commit adding only the `vendor` target to the Makefile.